playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -1,26 +1,29 @@
 ---
```

Comment:
Why is this still a file?

Reply:
We still need parameters to call the playbook.  How do you suggest we set those parameters?